### PR TITLE
Reduce ASLR entropy on ubuntu 24.04 for compatibility with TSAN in do…

### PR DESCRIPTION
**Pull request by jcristau**
…cker

https://bugzilla.mozilla.org/show_bug.cgi?id=1922578

### CODE REVIEW

**Comment by jwmoss** [comment]
It's not really clear and I'll work on contributing guides/documentation, but you'll need to add this script explicitly here: https://github.com/mozilla-platform-ops/worker-images/blob/main/gcp.pkr.hcl#L171 so that provisioner knows to run this shell script.

**Comment by jcristau** [comment]
Fixed, thanks!

File: gcp.pkr.hcl [hunk 1]
```diff
@@ -169,6 +169,7 @@ build {
       "${path.cwd}/scripts/linux/ubuntu-2404-amd64/fxci/01-bootstrap.sh",
       "${path.cwd}/scripts/linux/ubuntu-2404-amd64/fxci/02-additional-packages.sh",
       "${path.cwd}/scripts/linux/ubuntu-2404-amd64/fxci/03-hg.sh",
+      "${path.cwd}/scripts/linux/ubuntu-2404-amd64/fxci/04-aslr.sh",
```
Comment (jwmoss) [inline gcp.pkr.hcl:172]: You'll need to remove the `,` at the end! Otherwise looks good.

Comment (jcristau) [reply gcp.pkr.hcl:172]: Really?  03-hg.sh had the comma at the end, so presumably that worked?

Comment (jwmoss) [reply gcp.pkr.hcl:172]: Oh interesting. Let me merge it in and try it.

**Review by jwmoss — APPROVED** [review]

**Pull request merged**